apps/test-viewer/package.json

```diff
@@ -7,6 +7,7 @@
     "@itwin/appui-abstract": "^3.7.1",
     "@itwin/appui-layout-react": "^4.0.0-dev.15",
     "@itwin/appui-react": "^4.0.0-dev.15",
+    "@itwin/breakdown-trees-react": "workspace:*",
```

Comment:
take a look at the readme for the viewer

you need to add the depedendency meta

just like measure-tools